python/ray/tune/trial.py

```diff
@@ -13,6 +13,7 @@
 
 import ray
 import ray.cloudpickle as cloudpickle
+from ray.exceptions import GetTimeoutError
```

Comment:
Is there some way we can test this PR?

Reply:
We could include a test for this in one of the release tests using s3, or we could use a library like moto to mock s3, though that may take some effort to set up

Reply:
OK. How about let's keep track of this in an issue, as we should add multi-testing soon.

Reply:
Do you know why the release test set up cannot access S3?